streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -1473,13 +1475,15 @@ private void closeTaskDirty(final Task task, final boolean removeFromTasksRegist
             // before suspending and closing the topology
             task.prepareCommit();
         } catch (final RuntimeException swallow) {
-            log.error("Error flushing caches of dirty task {}", task.id(), swallow);
+            log.warn("Error flushing cache of dirty task {}. " +
+                "Since the task is closing dirty, the following exception is swallowed: ", task.id(), swallow);
         }
 
         try {
             task.suspend();
         } catch (final RuntimeException swallow) {
-            log.error("Error suspending dirty task {}: {}", task.id(), swallow.getMessage());
+            log.warn("Error suspending dirty task {}. " +
+                "Since the task is closing dirty, the following exception is swallowed: ", task.id(), swallow);
```

Comment:
Maybe it might be better to just log the exception message instead of the whole stacktrace.

WDYT?

Reply:
+1 to that

Reply:
OK, I replaced the stacktraces with the exception messages.